test/functional/http/post/4.multisignature.js

```diff
@@ -728,4 +728,17 @@ describe('POST /api/transactions (type 4) register multisignature', () => {
 			pendingMultisignatures
 		);
 	});
+
+	describe('verification', () => {
```

Comment:
What is the difference between this tests and one included in `4.multisignature.advanced.js` file?